src/Shared/NcdfUtil/hco_ncdf_mod.F90

```diff
@@ -413,6 +413,18 @@ SUBROUTINE NC_READ_TIME( fID,     nTime,        timeUnit, &
     ! Read calendar attribute
     IF ( PRESENT( timeCalendar ) ) THEN
        CALL NcGet_Var_Attributes( fId, v_name, 'calendar', timeCalendar )
+
+       ! Throw an error for climatological calendars without leap years
+       SELECT CASE( TRIM( v_name ) )
+          CASE( '360_day', '365_day', 'noleap' )
```

Comment:
Some other calendars to include are all_leap and 366_day.

Reply:
Thanks @lizziel. Now fixed in commit ba24290.